docs/versioned_docs/version-zenith/guides/snippets/github-google-cloud/main.yml

```diff
@@ -0,0 +1,22 @@
+name: 'ci'
+
+on:
+  push:
+    branches:
+    - master
```

Comment:
Most repos default to a `main` branch nowadays.

Reply:
Fixed